dist/add-markers.js

```diff
@@ -773,7 +811,7 @@ Fliplet.InteractiveMap.component('add-markers', {
     var _created = _babel_runtime_helpers_asyncToGenerator__WEBPACK_IMPORTED_MODULE_1___default()(
     /*#__PURE__*/
     _babel_runtime_regenerator__WEBPACK_IMPORTED_MODULE_0___default.a.mark(function _callee() {
-      var _this14 = this;
+      var _this17 = this;
```

Comment:
@squallstar Can you advise how `_this99999` can be handled. This seems like a terrible way to do this. I understand this is already an established structure based on the diff, but should probably be changed as it currently exists as a risk for error.

Reply:
@tonytlwu again, you are looking at the transpiled code. In the actual code of the component we only use `this`.

Reply:
@tonytlwu as @hcarneiro said, you should not look at files in the `dist/` folder.